### DISCUSSION TITLE
Welcome to simplyRetro-Z5 Discussions!

### DISCUSSION THREAD

**Discussion by geaz** [geaz/simplyRetro-Z5#17] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Clarion30** [comment]
Hello, nice project. Is the image also available for a Raspberry pi zero 2 w or just for the 1?
I would be happy if you would make another image for the Raspberry zero 2, thank you

**Reply by geaz** [reply]
Hey! Unfortunately I don't have a Zero2 at hand at the moment. Maybe in the future, but you should be able to use any other retro image out there. At the time I made this project, I created the custom image, because the other systems had high boot times. I think this should be way better with the Zero2 now (not tested).

**Comment by Clarion30** [comment]
Hello, I use the Z5 with Retropie and Zero2 and would like to try out the special image with the battery indicator, unfortunately I am not able to create an image
